src/main/java/de/tum/in/www1/artemis/service/SubmissionExportService.java

```diff
@@ -38,8 +32,14 @@
 
     private final ExerciseRepository exerciseRepository;
 
-    public SubmissionExportService(ExerciseRepository exerciseRepository) {
+    private final ZipFileService zipFileService;
+
+    private final FileService fileService;
+
+    public SubmissionExportService(ExerciseRepository exerciseRepository, ZipFileService zipFileService, FileService fileService) {
```

Comment:
Codacy found an issue: [Parameter 'exerciseRepository' is not assigned and could be declared final](https://app.codacy.com/gh/ls1intum/Artemis/pullRequest?prid=5968852)